install/kubernetes/helm/subcharts/mixer/templates/deployment.yaml

```diff
@@ -132,6 +146,14 @@
         - name: istio-certs
```

Comment:
Do we still mount istio-certs ?

Reply:
for this PR yes, I will disable secret mount(when sds is enabled) in a follow up PR(updating pilot/mixer bootstrap).